src/Essentials/samples/Samples/View/MediaPickerPage.xaml

```diff
@@ -16,7 +16,7 @@
             <Button Text="Capture video" Command="{Binding CaptureVideoCommand}" />
 
             <Image VerticalOptions="FillAndExpand" Source="{Binding PhotoPath}" IsVisible="{Binding ShowPhoto}" />
-            <MediaElement VerticalOptions="FillAndExpand" Source="{Binding VideoPath}" IsVisible="{Binding ShowVideo}" />
+            <!--<MediaElement VerticalOptions="FillAndExpand" Source="{Binding VideoPath}" IsVisible="{Binding ShowVideo}" />-->
```

Comment:
Note: I had to remove this because it is no longer in Maui, nad the XCT is not yet Maui.